Podfile

```diff
@@ -175,7 +175,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.81.1'
+  gutenberg commit: 'b4904076d045f99969d2a0dda6ae74630c8e1e95'
```

Comment:
Friendly reminder: we'll want to update this reference to the soon-to-be-created GB Mobile `1.82.2` tag.